epsagon/events/botocore.py

```diff
@@ -117,13 +117,82 @@ def update_response(self, response):
         :param response: Response from botocore
         :return: None
         """
-        self.event_id = response['ResponseMetadata']['RequestId']
+        event_id = response['ResponseMetadata'].get('RequestId')
+        if event_id:
+            self.event_id = event_id
         self.resource['metadata']['Retry Attempts'] = \
             response['ResponseMetadata']['RetryAttempts']
         self.resource['metadata']['Status Code'] = \
             response['ResponseMetadata']['HTTPStatusCode']
 
 
+class BotocoreCloudWatchEvent(BotocoreEvent):
+    """
+    Represents cloudwatch events (eventbridge) botocore event.
+    """
+
+    RESOURCE_TYPE = 'eventbridge'
+    RESOURCE_TYPE_UPDATE = 'events'
```

Comment:
not important but would rename to event (non plural)

Reply:
'events' is the aws resource name `boto3.client('events')...`